OpenRA.Mods.Common/ActorExts.cs

```diff
@@ -141,6 +141,10 @@ public static void NotifyBlocker(this Actor self, IEnumerable<CPos> positions)
 					return false;
 			}
 
+			// Occupied by any actor, including ally/friendly units
+			if (self.World.ActorMap.GetActorsAt(pos).Any())
```

Comment:
I need a developer's decision here.

I'd check for stances and if any friendly unit is standing on the cell, harvester will not mine from the cell. As a side effect, harvesters will not harvest from the cell with anti-vehicle mine planted. This is inconsistent with other units, where friendly units crush friendly mines. I can check for mine trait but that is only for Red Alert mod, which seem bad for me.

If the cell is occupied by an enemy unit, decide the cell can't be mined unless occupier is cloaked.
  - It is safer not to crush a hijacker or terrorists
  - A player can make AI/enemy player harvester get stuck if a stealth tank is deployed with Hold Fire stance on the resource field. This is the same behavior with the current bleed version. This decision is because of the mine, which is cloaked also.